@uportal/grid-carousel/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@uportal/grid-carousel",
+  "private": true,
```

Comment:
Is this private?
Won't we publish this to the `npm` registry in the way we do with other components?
Or do you have another delivery mechanism in mind?

Reply:
Fixed